lib/anyway/loaders.rb

```diff
@@ -75,3 +75,4 @@ def find(id)
 require "anyway/loaders/base"
 require "anyway/loaders/yaml"
 require "anyway/loaders/env"
+require "anyway/loaders/doppler"
```

Comment:
We shouldn't add this loader by default—it would break existing applications. Check out #108 for conditional loader activation example.